### ISSUE TITLE
CSVtoJSON-Agent truncates fields when there is a comma.  

### ISSUE THREAD

**Issue by venkateshkandasamy** [awslabs/amazon-kinesis-agent#95]
Hi,

I am using CSVtoJSON data processing option before streaming data with comma as delimiter. I am encountering an issue where Agent truncates values when there are commas in values. Any ideas on how to solve this problem.

The following sample data "xxx, xxx.","xxx&xx" is being converted to,
"Field1":"xxx",
"Field2":" xxx.",

**Comment by chaochenq** [comment]
yeah agent can only read the CSV file which is purely delimited by comma, not quotes. In your case, you are expecting each column to be surrounded by quotes, and delimited by comma.

Unfortunately Agent is not smart enough to strip the quotes yet.
